code/game/machinery/ammolathe.dm

```diff
@@ -0,0 +1,64 @@
+/obj/machinery/r_n_d/fabricator/mechanic_fab/autolathe/ammolathe
+	name = "\improper Ammolathe"
+	desc = "Produces guns, ammunition, and firearm accessories."
+	icon_state = "ammolathe"
+	icon_state_open = "ammolathe_t"
+	nano_file = "ammolathe.tmpl"
+	default_mat_overlays = TRUE
+
+	start_end_anims = 1
+
+	build_time = 0.5
+
+	allowed_materials = 0
```

Comment:
What would this do?

Reply:
allowed_materials is a var for what the name implies. The autolathe limits it to glass and metal but setting it to false/0/etc removes restrictions inherited from the autolathe to allow all mat types.

Reply:
Ooh. I like it